webapp/src/scriptsearch.tsx

```diff
@@ -388,6 +388,7 @@ export class ScriptSearch extends data.Component<ISettingsProps, ScriptSearchSta
         return (
             <sui.Modal isOpen={this.state.visible} dimmer={true}
                 className="searchdialog" size="fullscreen"
+                hideGoBack={!!this.state.resolve}
```

Comment:
Can we not use the closeIcon={false} to indicate that we don't want a go back button?

Reply:
sure can